README.md

```diff
@@ -120,6 +120,48 @@ Launch the application with the following commands:
 
 ![CloudWatchMetrics01.png](docs/images/BookstoreRVizPlan01.png)
 
+## Run with a WorldForge world
+
+After exporting a world from WorldForge, we can unzip the content and move under simulation_ws package:
+
+```bash
+unzip exported_world.zip
+mv aws_robomaker_worldforge_pkgs simulation_ws/src/
+```
+
+Build it again
+
+```bash
+cd simulation_ws
+colcon build
+```
+
+Launch the application with the following commands:
+
+```bash
+source simulation_ws/install/local_setup.sh
+roslaunch cloudwatch_simulation worldforge_turtlebot_navigation.launch
```

Comment:
this would not run without a map correct?

Reply:
I think this won't be able to load the map but it will not crash.

Reply:
That's not an ideal behavior right? let's put map generation as pre-requisite to navigation?

Reply:
I actually agree and think that's a better approach but that would make map generation NOT optional. I think it's reasonable to do that in another PR.

Reply:
Does map generation require that `genmap_script.sh` file? My understanding is that that script came from some instructions somewhere. Let's just include a short blurb in this README, pointing to those instructions.

In the future pull request, we'll add the `genmap_script.sh` file and relevant updates to this README, while possibly removing the pointer to those instructions.

Reply:
`genmap_script.sh` basically automates whatever is under `Generate map for your world` now.

Reply:
It looks like the argument here is that `worldforge_turtlebot_navigation.launch` will not crash with or without a map. If launched without a map, there will be no navigation (the TurtleBot won't be move) and that won't be ideal?

Let's just treat map generation as non-optional (no matter how user unfriendly it is to generate a map) and have `worldforge_turtlebot_navigation.launch` represent launching with a map present and navigation on.

Reply:
Yes^^. There is not much point in running worldforge_turtlebot_navigation is it doesn't lead to the inteded behavior of navigation. @mm318's recommended approach sounds good to me.

Reply:
Let's update the README with that approach. Hopefully that makes it easier. Less use cases = more simplicity.

Reply:
Makes sense, let's go with @mm318 suggestion. @ojasjoshi Would you mind making the changes? It includes making the map generation plugin NOT optional and I believe you have a better approach on how to organize the README

Reply:
Sure